lib/locales/es.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
         'Click to enter Plot title': 'Introducir el título de la Gráfica', // plot_api/plot_api.js:579
         'Click to enter X axis title': 'Introducir el título del eje X', // plots/plots.js:301
         'Click to enter Y axis title': 'Introducir el título del eje Y', // plots/plots.js:302
+        'Click to enter radial axis title': 'Introducir el título del eje radial',
```

Comment:
Just nit-picking: here I've translated the original `radial axis`, but I think `polar axis` is more common.

Reply:
Isn't `polar axis` ambiguous between angular and radial? Anyway that's a separate discussion, within the existing context this looks great, thanks!

Reply:
Oh, I've just realised that `radial axis` here is not the actual axis (i.e. the polar axis used as a reference to measure the angular coordinate).